assets/large_language_models/prompts/chat_hiking_chatbot/prompt/prompt.yaml

```diff
@@ -26,9 +26,19 @@ messages:
 input_variables:
   - name: User_Location
     description: Location that the user is looking to hike
-    defaultValue: United States
+    default: United States
```

Comment:
Can you bump asset version?  The we can re-publish with the new version.

Reply:
done